docs/1-qms/0-quality-manual-policy-objectives.md

```diff
@@ -97,17 +117,24 @@ comparisons are undergone utilising the following metrics:
 > a goal-oriented system in place to track your team's work? Even better: align
 > business and quality goals and describe your system here. -->
 
+Software development goal tracking is managed through the GitHub issue system.
+See <https://github.com/RadiotherapyAI/rai/issues>.
+
 ## 3. Roles
 
 <!-- > Describe the roles of the people in your company. Typically this is done by
 > drawing an organigram (you could use draw.io for that). Or, you just use a
 > table like below. -->
 
-| Role(s)                                     | People           |
-| ------------------------------------------- | ---------------- |
-| CEO, QMO, PRRC, CTO, CPO, Software Engineer | Simon Biggs      |
-| Regulatory Consultant                       | Dr Oliver Eidel  |
-| Regulatory Consultant                       | Pierre Lonchampt |
+| Role(s)               | People           |
+| --------------------- | ---------------- |
+| CEO                   | Simon Biggs      |
+| QMO                   | Simon Biggs      |
+| PRRC                  | Simon Biggs      |
+| CTO                   | Simon Biggs      |
+| Advisor               | Stuart Swerdloff |
```

Comment:
very thoughtful of you!

Reply:
I wanted to check, is this okay with you?

Reply:
Definitely.

Reply:
😄